com.github.scrivanolabs.scrivano.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.scrivanolabs.scrivano
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: Scrivano
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-config
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
+  - --device=dri
+modules:
+  - name: Scrivano
+    buildsystem: simple
+    build-commands:
+    - install -D Scrivano /app/bin/Scrivano
+    - install -D libskia.so /app/lib/libskia.so
+    - install -D libpdfium.so /app/lib/libpdfium.so
+    - install -Dm644 com.github.scrivanolabs.scrivano.appdata.xml -t /app/share/metainfo
+    - install -Dm644 com.github.scrivanolabs.scrivano.desktop -t /app/share/applications
+    - install -Dm644 app_icon.svg /app/share/icons/hicolor/scalable/apps/com.github.scrivanolabs.scrivano.svg
+    sources:
+    - type: archive
+      url: https://github.com/scrivanolabs/ScrivanoForLinux/releases/download/0.13.4/Scrivano_flatpak_files_0.13.4.zip
+      sha256: b1d5730851b2bfdc5f69c6914c6ce1c21a48ac8e1d75a81e09b8ea9383c77959
+      only-arches:
+        - 'x86_64'
```

Comment:
not this doesn't do what you think it does.

https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

It remain disappointing though.

Reply:
Ah I see, thank you for pointing this out. I wasn't aware of the need of a separate file to specify build architectures.